### ISSUE TITLE
GraphQL 0.10.5 Breaks

### ISSUE THREAD

**Issue by ezweave** [howtographql/howtographql#296]
Installing on a new machine, I ran across this (as of commit 0000023) message:

`Schema must be an instance of GraphQLSchema. Also ensure that there are not multiple versions of GraphQL installed in your node_modules directory.`

Deleting `node_modules` and locking the version to 0.10.3, then running `yarn install` appears to fix it.  e.g. editing `package.json` so that `graphql: 0.10.3` locks the version.  I'm going to see if I can track down the breaking change, as we're trying to use this as an educational tool within our company.

**Comment by angly-cat** [comment]
Do you mean you can't start the site locally? If so, I kinda was able to run it without locking `graphql` version.

**Comment by ezweave** [comment]
Yup.  I haven't checked since I posted this, but (at the time), that was the only way to get it to run.

**Comment by angly-cat** [comment]
Could you recheck? And if it still broken for you, specify your `node` and `yarn` versions.

P.S. couple of weeks ago the only option to successfully run the site locally was to lock `graphql` to version `0.11.7`.
